src/NuGet.Core/Test.Utility/ProjectManagement/TestNuGetProjectContext.cs

```diff
@@ -25,7 +25,7 @@ public FileConflictAction ResolveFileConflict(string message)
             return FileConflictAction.IgnoreAll;
         }
 
-        public PackageExtractionContext PackageExtractionContext { get; set; } = new PackageExtractionContext();
+        public PackageExtractionContext PackageExtractionContext { get; set; } = new PackageExtractionContext(null);
```

Comment:
Done

---
In reply to: [60083124](https://github.com/NuGet/NuGet.Client/pull/504#discussion_r60083124) [](ancestors = 60083124)